cdk/lib/__snapshots__/manage-frontend.test.ts.snap

```diff
@@ -0,0 +1,1480 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@rBangay does this file need to be committed or is it auto-generated?

Reply:
Hey Paul, yep this needs to be committed, it's a snapshot of the current configuration and used to test if anything has changed.